.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/node:10-browsers
+    steps:
+      - checkout
+      - run: npm config set prefix "$HOME/.local"
+      - run: npm i -g origami-build-tools@^7
```

Comment:
This looks like the old circleci config

Reply:
I must be rocking an old version of obt, thanks I'll update